app/views/subnets/import.html.erb

```diff
@@ -1,19 +1,20 @@
 <% title _("Import IPv4 subnets") %>
 
-<h2><%= _("The following IPv4 subnets have been found") %></h2>
-
+<h2><%= _("The following IPv4 subnets have been found. Please review the details before creating them.") %></h2>
```

Comment:
Could you drop this to regular paragraph text, or perhaps try h3 etc? The text is very nearly the same as the main title.